client/src/error.rs

```diff
@@ -247,18 +247,11 @@ pub enum Error {
     #[error("can't burn and transition an output at the same time, chain ID: {0}")]
     BurnAndTransition(ChainId),
 
-    //////////////////////////////////////////////////////////////////////
-    // Participation
-    //////////////////////////////////////////////////////////////////////
-    /// Invalid participations error
+    /// Participation error
     #[cfg(feature = "participation")]
-    #[error("invalid participations")]
-    InvalidParticipations,
-    /// IO error
-    #[cfg(feature = "participation")]
-    #[error("`{0}`")]
+    #[error("{0}")]
     #[serde(serialize_with = "display_string")]
-    Io(#[from] std::io::Error),
+    Participation(#[from] iota_types::api::plugins::participation::Error),
```

Comment:
Maybe rename to ParticipationApi ?

Reply:
But it's not really from the API, it's only from the Participations struct

Reply:
One could argue that the types are part of the API but alright, no probs